packages/teleport/src/services/auth/auth.js

```diff
@@ -21,6 +21,14 @@ import cfg from 'teleport/config';
 import makePasswordToken from './makePasswordToken';
 
 const auth = {
+  u2fBrowserSupported() {
+    if (window.u2f) {
+      return null;
+    }
+
+    return new Error('Your browser does not support FIDO U2F protocol.');
```

Comment:
our errors are lower case without `.`

`your browser does not support FIDO U2F protocol`

Reply:
revised